packages/docs/PropTables/HeaderPropTable.tsx

```diff
@@ -11,34 +11,59 @@ const headerProps: Prop[] = [
   },
   {
     name: 'actions',
-    types: 'Array<ActionButtonProps | ActionDropdownProps>',
+    types: (
+      <>
+        {'Array<'}
+        <NextLink href={{ hash: 'action-button-prop-table', query: { props: 'action-button' } }}>
+          ActionButton
+        </NextLink>
+        {' | '}
+        <NextLink
+          href={{ hash: 'action-dropdown-prop-table', query: { props: 'action-dropdown' } }}
+        >
+          ActionDropdown
+        </NextLink>
+        {'>'}
```

Comment:
We actually haven't done this before, but I figured it was a better approach than our previous iterations.